team_red: ["ssss","rrrr"]
team_green: ["acastro","wwww"]
team_blue: ["qqq", "yyyy"]